base_rest/apispec/rest_method_param_plugin.py

```diff
@@ -31,30 +31,41 @@ def operation_helper(self, path=None, operations=None, **kwargs):
         if not operations:
             return
         for method, params in operations.items():
-            input_param = routing.get("input_param")
-            output_param = routing.get("output_param")
-            if input_param and isinstance(input_param, RestMethodParam):
-                parameters = params.get("parameters", [])
-                # add default paramters provided by the sevice
-                parameters.extend(self._default_parameters)
-                if method == "get":
-                    # get quey params from RequestMethodParam object
-                    parameters.extend(
-                        input_param.to_openapi_query_parameters(self._service)
-                    )
-                    # sort paramters to ease comparison into unittests
-                else:
-                    # get requestBody from RequestMethodParam object
-                    request_body = params.get("requestBody", {})
-                    request_body.update(
-                        input_param.to_openapi_requestbody(self._service)
-                    )
-                    params["requestBody"] = request_body
-                parameters.sort(key=lambda a: a["name"])
+            parameters = self._generate_pamareters(routing, method, params)
+            if parameters:
                 params["parameters"] = parameters
-            if output_param and isinstance(output_param, RestMethodParam):
-                responses = params.get("responses", {})
-                # get response from RequestMethodParam object
-                responses.update(self._default_responses.copy())
-                responses.update(output_param.to_openapi_responses(self._service))
+            responses = self._generate_response(routing, method, params)
```

Comment:
```suggestion
            responses = self._generate_responses(routing, method, params)
```
or, rename var to `response`? Do you expect multiple items?

Reply:
responses is a list... I'll rename